appveyor.yml

```diff
@@ -71,7 +71,7 @@ install:
       ls $destination
 
   # Upgrade to the latest pip.
-  - '%CMD_IN_ENV% python -m pip install -U pip setuptools wheel'
+  - '%CMD_IN_ENV% python -m pip install -U pip "setuptools<50.0" wheel'
```

Comment:
I would go with `!=50.0.0` or so, such that when the patch/fix is released we get it

Reply:
agreed, `<50.0` is nicer.